integration-cli/docker_cli_external_graphdriver_unix_test.go

```diff
@@ -349,12 +349,12 @@ func (s *DockerExternalGraphdriverSuite) TestExternalGraphDriver(c *check.C) {
 }
 
 func (s *DockerExternalGraphdriverSuite) testExternalGraphDriver(name string, ext string, c *check.C) {
-	s.d.StartWithBusybox(c, "-s", name)
+	s.d.StartWithBusybox(c, "-s", name, "--experimental")
```

Comment:
I think I ran into this because in order for this test to work for me, I have to manually run the make script (whereas I normally have a helper bash function) and I didn't set `DOCKER_EXPERIMENTAL=1` in doing so.
Looks good to remove this when setup properly ;)